include/circt/Dialect/FIRRTL/FIRRTLTypes.td

```diff
@@ -99,6 +99,13 @@ def RWProbe : FIRRTLDialectType<
   CPred<"type_isa<RefType>($_self) && type_cast<RefType>($_self).getForceable()">,
    "rwprobe type", "::circt::firrtl::RefType">;
 
+def RWProbeTarget : FIRRTLDialectType<
+    CPred<"type_isa<FIRRTLBaseType>($_self) && "
+          "!type_cast<FIRRTLBaseType>($_self).hasUninferredReset() &&"
+          "!type_cast<FIRRTLBaseType>($_self).hasUninferredWidth()">,
+    "a rwprobe-able type (base, no uninferred widths or resets)",
```

Comment:
Nit: the better, verbose wording would help with the somewhat crappier error message now: "`RWProbeTarget (a FIRRTL base type with a known width and not abstract reset)`. This is mixing both actionable developer info ("RWProbeTarget") and actionable user info ("what this actually is").